compare_outputs.py

```
@@ -103,11 +137,24 @@ def classify_file(args):
+    baseline_lines = baseline_text.splitlines()
+    after_lines = after_text.splitlines()
+
+    if len(baseline_lines) != len(after_lines):
+        # Line count changed — fall back to full normalization
+        if VOLATILE_RE.sub('', baseline_text) == VOLATILE_RE.sub('', after_text):
+            return ('timestamp_only', rel_path)
+        return ('content_diff', rel_path)
 
-    return ('content_diff', rel_path)
+    # Same line count: only normalize the lines that differ
+    for b_line, a_line in zip(baseline_lines, after_lines):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)